dags/acoustic/acoustic_client.py

```diff
@@ -0,0 +1,188 @@
+"""
```

Comment:
Should this be in the `dags/` directory? No strong opinion but this seems like `utils/`

Reply:
moved the folder inside existing folder: `dags/utils/`